backend/src/test/java/gov/cdc/usds/simplereport/api/ResourceLinks.java

```diff
@@ -8,4 +8,7 @@
 
   public static final String WAITLIST_REQUEST = "/account-request/waitlist";
   public static final String ACCOUNT_REQUEST = "/account-request";
+  public static final String USER_ACCOUNT_REQUEST = "/user";
+  public static final String USER_SET_PASSWORD = "/user/set_password";
```

Comment:
We seem to be using kebab-case as our "standard' for multiword URL components, we should probably stick with that rather than having a mix of kebab-case and snake_case.